### PR TITLE
Added Anime News Network and Akinator

### PR DESCRIPTION
**Pull request by sunx2**
it shows almost all presences. if i missed something, i will update them.
It's made with for the presence request https://github.com/PreMiD/Presences/issues/301

image
image
image
image
image

### CODE REVIEW

File: Anime News Network/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "url": "www.animenewsnetwork.com",
+    "version": "1.1",
+    "logo": "https://cdn.discordapp.com/attachments/270798717089480704/631523459268345856/66183238-75d14f80-e64e-11e9-9409-f2ca1bbd3d0b.png",
+    "thumbnail": "https://cdn.discordapp.com/attachments/270798717089480704/631523654509002752/unknown.png",
+    "color":"#afeeee",
+    "tags": [
+        "Anime",
+        "News"
+    ],
+    "category": "other"
```
Comment (Bas950) [inline Anime News Network/dist/metadata.json]: Change category to anime

Comment (sunx2) [reply Anime News Network/dist/metadata.json]: ok

Comment (sunx2) [reply Anime News Network/dist/metadata.json]: Done

**Comment by sunx2** [comment]
screenshots for akinator

image
image

image
The selecting shows current mouseover option.

**Pull request merged**